htdocs/search-item.php

```diff
@@ -130,8 +130,7 @@
                             } else {
                                 $query = "SELECT *
                                           FROM bid
-                                          WHERE bidder = '" . $_SESSION['usr_email'] . "'
-                                          AND item_id = '{$itemId}'";
```

Comment:
Dont we need this to check if he has bidded for this particular item? If you remove this, it will just check if he has bidded before on anything.